src/Physics/Plugins/ammoJSPlugin.ts

```diff
@@ -578,8 +769,13 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
      * @param velocity velocity to set
      */
     public setLinearVelocity(impostor: PhysicsImpostor, velocity: Vector3) {
-        this._tmpAmmoVectorA.setValue(velocity.x, velocity.y, velocity.z);
-        impostor.physicsBody.setLinearVelocity(this._tmpAmmoVectorA);
+        if (!impostor.soft) {
+            this._tmpAmmoVectorA.setValue(velocity.x, velocity.y, velocity.z);

```

Comment:
I see a lot of these are not supported, is this a limitation of the engine or something that could be added in the future?

Reply:
For some reason I thought they were not supported for soft bodies. When I went back to check I find they are. Will correct.

Reply:
Understand know why I thought they were not supported. For rigid bodies you use getLinearVelocity and setLinearVelocity whereas it turns out for soft bodies you use linearVelocity. Same for angular. Have now combined them in the plugin

Reply:
Done